.github/workflows/release.yaml

```diff
@@ -44,7 +44,6 @@ jobs:
       - name: Upload charm to charmhub
         uses: canonical/charming-actions/upload-charm@2.1.0
         with:
-          resource-overrides: "pgbouncer-image:3"
```

Comment:
Is [this](https://github.com/canonical/pgbouncer-k8s-operator/actions/runs/3846492286/jobs/6553180944#step:4:23) still a problem?

Reply:
I think overriding pgbouncer-image is unnecessary, since we can just pull it from dockerhub